lms/urls.py

```diff
@@ -359,6 +359,10 @@
             'instructor.views.coupons.update_coupon', name="update_coupon"),
         url(r'^courses/{}/get_coupon_info$'.format(settings.COURSE_ID_PATTERN),
             'instructor.views.coupons.get_coupon_info', name="get_coupon_info"),
+        url(r'^courses/{}/look_up_registration_code$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
It would be ideal if these registration code URLs were name-spaced. At a minimum, would you mind cleaning up the duplicated `courses/{}/` prefix?

```python
course_urls = patterns(
  '',
  url(r'^look_up_registration_code$', 'instructor.views.registration_codes.look_up_registration_code', name='look_up_registration_code'),
  ...
)

url(r'^courses/{}/'.format(settings.COURSE_ID_PATTERN), include(course_urls)),
```